kernel/kernel-api/src/main/java/io/delta/kernel/Snapshot.java

```diff
@@ -98,4 +99,10 @@ public interface Snapshot {
 
   /** @return a committer that owns and controls commits to this table */
   Committer getCommitter();
+
+  /**
+   * @return a {@link UpdateTableTransactionBuilder} to build an update table transaction
+   * @since 3.4.0
```

Comment:
nit: since 4.1.0

Reply:
I think this will release next with 3.4.0. I wish there was a better way to make sure to update this before next release

Reply:
Oh but then it's not in 4.0.0 :/